### ISSUE TITLE
Is it possible to use fn-fx as a front end of a mainly java based application

### ISSUE THREAD

**Issue by dmg46664** [halgari/fn-fx#8]
Possibly a clojure question, but I assume it's possible to use this as a template language in a java application?

http://clojure.org/reference/java_interop

maybe? Pardon my unfamiliarity with closure.

**Comment by blak3mill3r** [comment]
Yes, you certainly could do that. Clojure comes with excellent Java interop (both ways). If you have an existing Java application, you can embed Clojure and fn-fx in it (see Alex's answer to http://stackoverflow.com/questions/2181774/calling-clojure-from-java), and the rest works just like you'd expect.
